### ISSUE TITLE
Use TC'S code for Automatic Updates

### ISSUE THREAD

**Issue by Talmeron** [Arkania/ArkCORE-NG#42]
There HAS to be a way to use the code and edit it to point to this repo. Given how often you update, this would save a lot of trouble. As long as naming conventions are adhered.

**Comment by gpn39f** [comment]
what for problem you have?
you have every month a snapshot for the world table. its fine for 99.9% of use.. the world table has no stored character info: overwrite you world with newest snapshot. exchange the server files.. all is done..
if you really need newest update, up to last commit.. import the update folder from repository..
the update folder, has the packet zip from last month..(not needed, if actual snapshot is used..) and all updates for this month.. (note: this month is was really many.. so its first time I have stored first part of updates of December, in a _2015_12.zip)

**Comment by Talmeron** [comment]
No....not a problem. Just using the TC code to automatically get new updates for quests and stuff.....for in the future.

**Comment by Dokman** [comment]
this should be a good tool, so the server will know when it has to be updated by his self and will do it automatically

**Comment by Lbniese** [comment]
Is this something that's being looked into?
